### PR TITLE
Pass down wallet to Boost component

### LINKED ISSUE #1778: Boosting Listings
It seems that step 3 of listing creation is not working as intended. It currently does not recognize that a user is holding OGN and give them the appropriate interface for setting the boost level.

### PR DESCRIPTION
**Pull request by shahthepro**
Fixes #1778

### CODE REVIEW

File: dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js [hunk 1]
```diff
@@ -5,13 +5,19 @@ import Details from './Details'
...
 
 const FractionalHourlyListing = props => (
   <Steps
     {...props}
     steps={[
       { step: 1, component: Details, require: 'subCategory' },
-      { step: 2, component: Boost, path: 'boost', require: 'title' },
+      {
+        step: 2,
+        component: withWallet(Boost),
```
Comment (nick) [inline dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js]: I think we can wrap the Boost component itself before it's exported, similar to how we do it [here](https://github.com/OriginProtocol/origin/blob/master/dapps/marketplace/src/pages/messaging/Messages.js#L150)

File: dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js [hunk 2]
```diff
@@ -5,13 +5,19 @@ import Details from './Details'
 import Boost from './Boost'
 import Availability from './Availability'
 import Review from './Review'
+import withWallet from '../../../../hoc/withWallet'
```
Comment (nick) [inline dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js]: the `hoc` dir has an alias in webpack, so you can import it with just `import withWallet from 'hoc/withWallet'`

File: dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js (hunk 1 above)
Comment (shahthepro) [reply dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js]: Done.

File: dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js (hunk 2 above)
Comment (shahthepro) [reply dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js]: Didn't know about the alias. Have updated it now.

**Comment by nick** [comment]
Thanks for this... would you mind pushing your branch direct to this repo now that you're a collaborator? Then the tests will run automatically.

**Comment by shahthepro** [comment]
Opened another PR with same changeset https://github.com/OriginProtocol/origin/pull/1783

**Pull request closed without merging**